lib/appsignal.ex

```diff
@@ -1,3 +1,5 @@
+Code.require_file("agent.ex")
```

Comment:
Not sure if this is the best way to require this file?
It may be better than eval-ing the file